userguide/jails.rst

```diff
@@ -307,30 +305,30 @@ these options.
    |                       |           |                                                                                                                     |
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
    | exec.system_jail_user | string    | Set this boolean option to *True* to look for the :guilabel:`exec.jail_user` in the system                          |
```

Comment:
I think the text should not use guilabel, but should just say "to look for the jail user in the system".

But it's not clear, please verify.

Reply:
So it is an actual field in the jail properties tab of "Advanced Jail Creation" on the FreeNAS system. How would you like to go about it?